src/types.ts

```diff
@@ -17,6 +17,33 @@ export interface DocProps {
     revision?: string | null;
 }
 
+export interface TableData {
+    columnMetadata: ColumnMetadata[];
+    data: string[][];
+    columnwidth?: number;
+}
+
+export interface ColumnMetadata {
+    name: string;
+    type: number;
```

Comment:
don't you want it to be from type dataTypes?

Reply:
No because dataTypes is just a number that represents the type. If I use it as a type here I will get an array of datatype enum, instead of an array of string/number/boolean values.